src/components/overview/LeftPanel.tsx

```diff
@@ -69,95 +88,91 @@ export default function LeftPanel({ onOpenSettingsModal }: Props) {
       justifyContent="space-between"
     >
       <VStack w="full" h="full" overflowY="auto">
-      <VStack w="full" h="fit-content" p={4} gap={2}>
-        <HStack w="full" justifyContent="space-between" px={1}>
+        <VStack w="full" h="fit-content" p={4} gap={2}>
           <Text w="full" fontWeight="bold" fontSize="28px">
             Collections
           </Text>
-          <Tooltip
-            hasArrow
-            label={
-              isMacroOutputEnabled
-                ? 'Disable Macro Output'
-                : 'Enable Macro Output'
-            }
-            closeOnClick={false}
-            variant="brand"
-          >
-            <Box>
-              <Switch
-                size="sm"
-                variant="brand"
-                defaultChecked={isMacroOutputEnabled}
-                isChecked={isMacroOutputEnabled}
-                aria-label="Macro Output Toggle"
-                onChange={() => {
-                  setIsMacroOutputEnabled((value) => {
-                    updateMacroOutput(value).catch((e) => {
-                      error(e)
-                      toast({
-                        title: `Error ${
-                          !value ? 'disabling' : 'enabling'
-                        } macro output`,
-                        description: `Unable to ${
-                          !value ? 'disable' : 'enable'
-                        } macro output, please re-open the app. If that does not work, please contact us on Discord.`,
-                        status: 'error',
-                        duration: 2000,
-                        isClosable: true
-                      })
-                    })
-                    return !value
-                  })
-                }}
+
+          <SearchBar />
+          <Divider />
+        </VStack>
+        <VStack
+          w="full"
+          h="full"
+          overflowY="auto"
+          overflowX="hidden"
+          ref={parent}
+          px={4}
+          spacing={1}
+          sx={useScrollbarStyles()}
+        >
+          {searchValue.length === 0 &&
+            collections.map((collection: Collection, index: number) => (
+              <CollectionButton
+                collection={collection}
+                index={index}
+                key={`${collection.name} + ${index}`}
+                isFocused={index == selection.collectionIndex}
+                isMacroOutputEnabled={isMacroOutputEnabled}
+                setFocus={(index) => changeSelectedCollectionIndex(index)}
+                toggleCollection={() =>
+                  onCollectionUpdate(
+                    {
+                      ...collections[index],
+                      active: !collections[index].active
+                    },
+                    index
+                  )
+                }
               />
-            </Box>
-          </Tooltip>
-        </HStack>
-        <Divider />
+            ))}
+          {searchValue.length === 0 && (
```

Comment:
It might be a good idea to generalise this logic into "isSearching" and use that variable here and above. Then you should also probably do `searchValue.trim().length` so you ignore just whitespace in the search

Reply:
If the searchValue tracking remains in the Application context, then I'd maybe put the `isSearching` value in the application context, so a lot of these places can just use that variable rather than having to determine it themselves

Reply:
Rewrote the logic a bit: [7fcc1d4](https://github.com/WootingKb/wooting-macros/pull/194/commits/7fcc1d4b167343ebc5386440a57ec7e1d003917c)